server/server.js

```diff
@@ -3,16 +3,54 @@ const bodyParser = require('body-parser');
 require('dotenv').config();
 
 const app = express();
+const http = require('http').Server(app);
+const io = require('socket.io')(http, {
+  cors: {
+    origin: 'http://localhost:3000', // this likely needs to be changed to wherever the origin requests are coming from
+    methods: ['GET', 'POST'],
+  },
+});
 
 const sessionMiddleware = require('./modules/session-middleware');
 const passport = require('./strategies/user.strategy');
 
 // Route includes
 const userRouter = require('./routes/user.router');
+const awsRouter = require('./routes/aws.router');
+const messageRouter = require('./routes/message.router');
+const vendorRouter = require('./routes/vendor.router');
+const searchRouter = require('./routes/search.router');
+const specialFeaturesRouter = require('./routes/specialFeaturesDD.router');
+const vendorTypesRouter = require('./routes/vendorTypesDD.router');
+
+
+// TODO AWS const
+const UploaderS3Router = require('react-dropzone-s3-uploader/s3router');
+
+// AWS Configuration //
+app.use(
+  '/s3',
+  UploaderS3Router({
+    bucket: 'cullen-contrive', // required
```

Comment:
Eventually you'll need to make this bucket an environment variable, so the client can configure this app to point at their own bucket on heroku.